docs/development-environment.md

```diff
@@ -68,7 +65,7 @@ $ pip install -e .
 
 ## NodeJS and modules
 
-You need NodeJS 6.2.4 (but your can check the version used in the `.nvmrc` file at the repository root)
+You need NodeJS but you should check the version used in the `.nvmrc` file at the repository root.
```

Comment:
Proposal (the "but" does not make sense to me): NodeJS is required to build or run the frontend. Please check the `.nvmrc` at the root of the repository to check the exact version of NodeJS you need.